docs/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Purpose
+
+Briefly describe the purpose of the change, and/or link to the JIRA ticket for context
+
+Fixes MLPAB-##
+
+## Approach
+
+Explain how your code addresses the purpose of the change
+
+## Learning
+
+Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Modernising LPA service
```

Comment:
if we're going to have a template, should probably not reference "MLPAB-" and "Modernising LPA service"

Reply:
thanks, updated